presto-hive/src/test/sql/create-test.sql

```diff
@@ -232,3 +232,8 @@ ALTER TABLE presto_test_partition_schema_change REPLACE COLUMNS (t_data DOUBLE);
 
 INSERT OVERWRITE TABLE presto_test_partition_schema_change_non_canonical PARTITION (t_boolean='0')
 SELECT 'test' FROM presto_test_sequence LIMIT 100;
+
+ANALYZE TABLE presto_test_unpartitioned COMPUTE STATISTICS;
+ANALYZE TABLE presto_test_unpartitioned COMPUTE STATISTICS FOR COLUMNS;
+ANALYZE TABLE presto_test_bucketed_by_string_int PARTITION(ds) COMPUTE STATISTICS;
+ANALYZE TABLE presto_test_bucketed_by_string_int PARTITION(ds) COMPUTE STATISTICS FOR COLUMNS;
```

Comment:
Does it run a map-reduce job in hadoop? Do you think it may cause some intermittent failures on travis? Which job it affects and how much longer it takes now to execute that job?